From 6255f1c2ee5b586bd41fc1caea73c57ccd3e1339 Mon Sep 17 00:00:00 2001
From: Felix Fietkau <nbd@nbd.name>
Date: Mon, 24 Jan 2022 13:16:00 +0100
Subject: [PATCH] scripts/feeds: fix installing targets without explicitly
 specifying the feed

Add similar code to what is done on packages

Signed-off-by: Felix Fietkau <nbd@nbd.name>
(cherry-picked from commit 0bdf8d12063b9be1cd73798f54a53279d21c9c95)
---
 scripts/feeds | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/feeds b/scripts/feeds
index 9d66ea5d9888..007726c5d38c 100755
--- a/scripts/feeds
+++ b/scripts/feeds
@@ -499,6 +499,8 @@ sub install_target {
 	my $feed = shift;
 	my $name = shift;
 
+	$feed = lookup_target($feed, $name);
+
 	$installed_targets{$name} and return 0;
 
 	$feed = $feed_cache{$feed->[1]}->[2];
-- 
2.34.1

